enterprise/mqtt/src/main/java/io/crate/mqtt/netty/Netty4MqttServerTransport.java

```diff
@@ -97,6 +105,7 @@
     private final SslContextProvider sslContextProvider;
```

Comment:
You already have the context here. We just need to lazy load it on `get()` and then we're good without complicated load strategies.